### ISSUE TITLE
small object is not good?

### ISSUE THREAD

**Issue by CPFelix** [SimonZeng7108/efficientsam3#25]
Thanks for the great work!
When I use the pretrained model(like ES-RV-L ES-TV-L ES-TV-M-MC-S1),I found that when objects in the image are small,then the model seg nothing.
Is it the loss caused by distill and is normal? @SimonZeng7108 
test image:
--prompt  "car"
![Image](https://github.com/user-attachments/assets/4126f107-5ef2-49f2-9149-06f0d295c5c5)

**Comment by colinlin1982** [comment]
问题与 https://github.com/SimonZeng7108/efficientsam3/issues/17 相似。

目前可以生成稳定结果的蒸馏模型是litetext分支下的efficient_sam3_image_encoder_mobileclip_s1_ctx16，我用你这张图像（尺寸压缩到1/4）试了一下：

<img width="1636" height="1099" alt="Image" src="https://github.com/user-attachments/assets/e55ab314-5b64-428d-90b7-718c4fbc9fec" />

**Comment by SimonZeng7108** [comment]
Yes, unfortunately the efficient sam3 image encoders still underperforming at stage 1. For your problem, maybe it's worth finetuning on this dataset - [Tracking-Any-Granularity](https://github.com/MCG-NJU/SAM2-Plus).

**Comment by CPFelix** [comment]
Thanks for your reply! @colinlin1982 
I try your advice and the cars are seg out. But I still want seg "water" for this image, and I also found low conf question and wrong results(I change conf_threhold=0.1):
boxes 置信度:[0.1080, 0.1108, 0.1586, 0.1013]
<img width="1170" height="770" alt="Image" src="https://github.com/user-attachments/assets/1834357c-1bfd-41a5-9f83-77b7a3037bae" />
I think the problem is text_encoder lost some information about rare category(No coco category).
The official sam3(conf_threshold=0.4):
masks 数量: 2
  - boxes 数量: 2
  - boxes 置信度: tensor([0.5713, 0.4380]
![Image](https://github.com/user-attachments/assets/cb786753-78c0-4279-9db4-4758846d72b6)

**Comment by colinlin1982** [comment]
@CPFelix 我有个建议，只针对你这个应用场景，你可以试试提示词"muddy water"。

**Comment by CPFelix** [comment]
谢谢你的建议 @colinlin1982 我试了试确实能检测出了。看来类别的描述也很重要。

<img width="1170" height="767" alt="Image" src="https://github.com/user-attachments/assets/f5ba63a4-3ad0-4e9a-bf3e-d72f265034ae" />
